mobile/native/src/logger.rs

```diff
@@ -28,8 +28,8 @@ pub fn log_base_directives(env: EnvFilter, level: LevelFilter) -> Result<EnvFilt
         // set to debug to show ldk logs (they're also in logs.txt)
         .add_directive("sled=warn".parse()?)
         .add_directive("bdk=warn".parse()?) // bdk is quite spamy on debug
+        .add_directive("lightning=trace".parse()?)
```

Comment:
hey, I'm working on this ticket! ;D

Reply:
Good to know! I didn't mean to push that here anyway :D